neo/tools.cmake

```diff
@@ -8,6 +8,9 @@ file(GLOB COMPILER_AAS_SOURCES tools/compilers/aas/*.cpp)
 file(GLOB COMPILER_DMAP_INCLUDES tools/compilers/dmap/*.h)
 file(GLOB COMPILER_DMAP_SOURCES tools/compilers/dmap/*.cpp)
 
+file(GLOB CODEC_FONT_INCLUDES tools/compilers/fontcodec/*.h)
+file(GLOB CODEC_FONT_SOURCES tools/compilers/fontcodec/*.cpp tools/compilers/fontcodec/FontTool.cxx )
```

Comment:
As in one of my other comments remove "tools/compilers/fontcodec/FontTool.cxx" after you changed the file ending to cpp.